src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "string",
+             "default": null,
+             "description": "Channel name. If none, try to auto detect channel",
+             "name": "channel_name"
```

Comment:
@edersondisouza I encountered a problem about the "multi-channel"  with buffer reading. Below is the test fbp file. It should output both adc1 and adc2 values, but it just print adc1 two times, and there is no adc2 output, how to narrow down this issue?
#!/usr/bin/env sol-fbp-runner
adc1(iio/adc:iio_device="INT3495 0",buffer_size=64)
adc2(iio/adc:iio_device="INT3495 0",buffer_size=64,channel_name="in_voltage5")
timer(timer:interval=1000)

timer OUT -> TICK adc1 OUT -> IN padc1(console:prefix="adc1:")
timer OUT -> TICK adc2 OUT -> IN padc2(console:prefix="adc2:")